dts/arm/st/g0/stm32g0.dtsi

```diff
@@ -206,6 +206,32 @@
 			status = "disabled";
 		};
 
+		can {
```

Comment:
Better to define the can node in the since the stm32g0x0 has none

Reply:
Good catch, done